CMakeLists.txt

```diff
@@ -122,6 +123,7 @@ option(ENABLE_Einsums "Enables the Einsums tensor library" OFF)
 option(ENABLE_simint "Enables use of SIMINT two-electron integral library" OFF)
 option(ENABLE_gauxc "Enables use of GauXC density functional library" OFF)
 option(ENABLE_gdma "Enables Stone's GDMA multipole code (requires Fortran; can also be added at runtime)" OFF)
+option(ENABLE_IntegratorXX "Enables use of IntegratorXX density functional grids library" OFF)
```

Comment:
Do we actually use this anywhere?

Reply:
There's a dummy module (like Einsums), so we check it compiles, not just detects. GauXC uses it (unfortunately it has to build its own b/c it expects different include paths than the installed project). OpenOrbitalOptimizer uses it for testing (that's irrelevant to Psi4). So the answer is "not really", but there's addons we use less.